Mathlib/RingTheory/Localization/FractionRing.lean

```diff
@@ -188,6 +198,30 @@ such that `z = f x * (f y)⁻¹`. -/
 noncomputable def lift (hg : Injective g) : K →+* L :=
   IsLocalization.lift fun y : nonZeroDivisors A => isUnit_map_of_injective hg y
 
+section liftAlgHom
+
+variable [Algebra R A] [Algebra R K] [IsScalarTower R A K] [Algebra R L]
+  {g : A →ₐ[R] L} (hg : Injective g) (x : K)
+include hg
+
+/-- `AlgHom` version of `IsFractionRing.lift`. -/
+noncomputable def liftAlgHom : K →ₐ[R] L :=
+  { lift hg with
+    commutes' := by
+      intro r
+      change IsLocalization.lift _ (algebraMap R K r) = _
+      simp [IsScalarTower.algebraMap_apply R A K]
+  }
```

Comment:
Can this just come from `IsLocalization.liftAlgHom`?

Reply:
Let me try.

Reply:
Done.